docs/Modern-LocalStateManagement.md

```diff
@@ -0,0 +1,185 @@
+---
+id: local-state-management
+title: Local State Management
+---
+
+Relay can be used to read and write local data, and act as a single source of truth for *all* data in your client application.  
+The Relay Compiler fully supports client-side extensions of the schema, which allows you to define local fields and types.  
+
+Table of Contents:
+
+- [Extending the client schema](#extending-the-client-schema)
+- [Querying local state](#querying-local-state)
+- [Mutating local state](#mutating-local-state)
+- [Initial local state](#initial-local-state)
+
+## Extending the client schema
+
+To extend the server schema, create a new `.graphql` file inside your `--src` directory.  
+Let's call it `./src/clientSchema.graphql`.
+
+This schema describes what local data can be queried on the client.  
+It can even be used to extend an existing server schema.
+
+For example, we can create a new type called `Note`:
+
+```graphql
+type Note {
+  ID: ID!
+  title: String
+  body: String
+}
+```
+
+And then extend the server schema type `User`, with a list of `Note`, called `notes`.
+
+```graphql
+extend type User {
+  notes: [Note]
+}
+```
+
+## Querying local state
+
+Accessing local data is no different from querying your GraphQL server.  
+Here, we use a [QueryRenderer](./query-renderer) to get the current `User` via the `viewer` field, along with her name and the list of notes.
```

Comment:
You can replace “her” with “their” in English to make this sentence speak to anyone.